canard.c

```diff
@@ -307,7 +307,7 @@ void canardHandleRxFrame(CanardInstance* ins, const CanardCANFrame* frame, uint6
 
         if (rx_state == NULL)
         {
-            return;
+            return -CANARD_ERROR_RX_NO_STATE;
```

Comment:
This should be `CANARD_ERROR_RX_MISSED_START`, judging by the logic. A new RX state is added when we see the first frame of the transfer. If this is not the first frame and the RX state does not exist, that means that we either didn't see the first frame, or we couldn't allocate a new state because of lack of memory. Either way, it qualifies as a missed start.